client/my-sites/media-library/content.jsx

```diff
@@ -162,8 +165,8 @@ export class MediaLibraryContent extends Component {
 					upgradeNudgeName = 'plan-media-storage-error-video';
 					upgradeNudgeFeature = 'video-upload';
 					message = translate(
-						'%d file could not be uploaded because your site does not support video files. Upgrade to a premium plan for video support.',
-						'%d files could not be uploaded because your site does not support video files. Upgrade to a premium plan for video support.',
+						'%(occurrences)d file could not be uploaded because your site does not support video files. Upgrade to a %(planName)s plan for video support.',
+						'%(occurrences)d files could not be uploaded because your site does not support video files. Upgrade to a %(planName)s plan for video support.',
```

Comment:
since these strings would need to be re-translated anyway - maybe this is worth smoothing? 😄 

```suggestion
						'%(occurrences)d file could not be uploaded because your site does not support video files. Upgrade to the %(planName)s plan for video support.',
						'%(occurrences)d files could not be uploaded because your site does not support video files. Upgrade to the %(planName)s plan for video support.',
```